src/basic/selinux-util.c

```diff
@@ -376,13 +376,9 @@ int mac_selinux_get_child_mls_label(int socket_fd, const char *exe, const char *
 char* mac_selinux_free(char *label) {
 
 #if HAVE_SELINUX
-        if (!label)
-                return NULL;
-
-        if (!mac_selinux_use())
-                return NULL;
-
         freecon(label);
```

Comment:
hmm, so freecon() being a wrapper around free() is an implementation detail we should probably not code against. (I mean, ffs, why do they even have that dreaded call?)

However their man page nowadays documents that freecon() is a NOP on NULL, hence I think your patch is OK.